src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -71,36 +70,37 @@ public static HearingDetails buildHearingDetails(HearingWrapper wrapper, Referen
             .build();
     }
 
-    public static boolean shouldBeAutoListed(@Valid SscsCaseData caseData) {
-        // TODO Future Work
-        return !isCaseLinked(caseData);
-    }
-
     public static String getHearingType() {
         return SUBSTANTIVE.getHmcReference();
     }
 
     public static HearingWindow buildHearingWindow(@Valid SscsCaseData caseData, boolean autoListed) {
+        return HearingWindow.builder()
+                .firstDateTimeMustBe(getFirstDateTimeMustBe())
+                .dateRangeStart(getHearingWindowStart(caseData, autoListed))
+                .dateRangeEnd(null)
+                .build();
+    }
+
+    public static LocalDate getHearingWindowStart(@Valid SscsCaseData caseData, boolean autoListed) {
 
-        if (!autoListed || isBlank(caseData.getDwpResponseDate())) {
-            LocalDate dateRangeStart = LocalDate.now().plusDays(DAYS_TO_ADD_HEARING_WINDOW_TODAY);
-            return HearingWindow.builder()
-                    .dateRangeStart(dateRangeStart)
-                    .dateRangeEnd(null)
-                    .build();
+        LocalDate startDate = LocalDate.now().plusDays(DAYS_TO_ADD_HEARING_WINDOW_TODAY);
+        if (isBlank(caseData.getDwpResponseDate())) {
```

Comment:
See my refactor of this for 10260. Basically, check if isNotBlank, then do DwpResponseDate stuff, then default at the end to LocalDate.now().plusDays(DAYS_TO_ADD_HEARING_WINDOW_TODAY)